annotations/kind-annotations/src/main/java/io/dekorate/kind/manifest/KindManifestGenerator.java

```diff
@@ -33,21 +34,16 @@ public String getKey() {
     return KIND;
   }
 
-  @Override
-  protected void addDecorators(String group, KubernetesConfig config) {
-    super.addDecorators(group, config);
-    if (config.getPorts().length > 0) {
-      resourceRegistry.decorate(group, new AddServiceResourceDecorator(config));
-    }
-  }
-
   @Override
   public void generate(KubernetesConfig kubernetesConfig) {
     initializeRegistry(kubernetesConfig);
-    addDecorators(KIND, kubernetesConfig);
+    Optional<KindConfig> kindConfig = configurationRegistry.get(KindConfig.class);
+
+    super.addDecorators(KIND, new KubernetesConfigBuilder(kubernetesConfig)
```

Comment:
Why are you using `super.addDecorators`?

Reply:
do you mean the `super.` appendix? We don't need it, we can simply use `addDecorators`

Reply:
If it was that, I've already removed the `super.`.